### PR TITLE
add labels to headless service template

### PR DESCRIPTION
**Pull request by yk12OT**

### CODE REVIEW

**Comment by yk12OT** [comment]
hey @leochu - could you please review/approve while Talin is out? Thanks!

File: charts/keycloak/templates/headless-service.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {{- $highAvailability := gt (int .Values.keycloak.replicas) 1 -}}
+{{- $service := .Values.keycloak.service -}}
```
Comment (leochu) [inline charts/keycloak/templates/headless-service.yaml:2]: Why is this needed to add labels?

Comment (yk12OT) [reply charts/keycloak/templates/headless-service.yaml:2]: I guess we don't need it anymore!  I have updated the `verify_k8s_labels.py` verification script so it looks into labels on pods, not labels on services.

**Comment by yk12OT** [comment]
Closing this PR without merging. Found alternative way to solve the original problem: https://github.com/opendoor-labs/code/pull/15452
